Src/DependencyCollector/Shared/HttpDependenciesParsingTelemetryInitializer.cs

```diff
@@ -86,9 +89,95 @@ private bool TryParseAzureBlob(string host, string name, string url, out string
+                {
+                    ////
+                    //// Queue Service REST API: https://msdn.microsoft.com/en-us/library/azure/dd179363.aspx
+                    ////
+
+                    account = host.Substring(0, host.IndexOf('.'));
+
+                    string nameWithoutVerb = name;
+
+                    for (int i = 0; i < this.azureTableVerbPrefixes.Length; i++)
+                    {
+                        var verbPrefix = this.azureTableVerbPrefixes[i];
+                        if (name.StartsWith(verbPrefix, StringComparison.OrdinalIgnoreCase))
+                        {
+                            verb = name.Substring(0, verbPrefix.Length);
+                            nameWithoutVerb = name.Substring(verbPrefix.Length);
+                            break;
+                        }
+                    }
+
+                    var slashPrefixShift = nameWithoutVerb[0] == '/' ? 1 : 0;
+                    var idx = nameWithoutVerb.IndexOf('/', slashPrefixShift); // typically first symbol of the path is '/'
+                    tableName = idx != -1 ? nameWithoutVerb.Substring(slashPrefixShift, idx - slashPrefixShift) : nameWithoutVerb.Substring(slashPrefixShift);
+                    idx = tableName.IndexOf('(');
+                    tableName = idx != -1 ? tableName.Substring(0, idx) : tableName;
+
+                    result = true;
+                }
+            }
+
+            return result;
+        }
+
+        private bool TryParseAzureQueue(string host, string name, string url, out string account, out string verb, out string queueName)
+        {
+            bool result = false;
+
+            account = null;
+            verb = null;
+            queueName = null;
+
+            if (name != null && host != null && url != null)
+            {
+                if (host.EndsWith(".queue.core.windows.net", StringComparison.OrdinalIgnoreCase))
+                {
+                    ////
+                    //// Queue Service REST API: https://msdn.microsoft.com/en-us/library/azure/dd179423.aspx
+                    ////
+
+                    account = host.Substring(0, host.IndexOf('.'));
+
+                    string nameWithoutVerb = name;
+
+                    for (int i = 0; i < this.azureQueueVerbPrefixes.Length; i++)
+                    {
+                        var verbPrefix = this.azureQueueVerbPrefixes[i];
+                        if (name.StartsWith(verbPrefix, StringComparison.OrdinalIgnoreCase))
+                        {
+                            verb = name.Substring(0, verbPrefix.Length);
+                            nameWithoutVerb = name.Substring(verbPrefix.Length);
+                            break;
+                        }
+                    }
+
                     var isFirstSlash = nameWithoutVerb[0] == '/' ? 1 : 0;
                     var idx = nameWithoutVerb.IndexOf('/', isFirstSlash); // typically first symbol of the path is '/'
```

Comment:
nitpick: I would still really like to be able to read "bool" and "int" instead of "var".